Debugging types errors in fastai

Let's say you run into this:

RuntimeError: Could not infer dtype of numpy.int8





It get really sensitive about data types


If you get this, try to reproduce it in Google Colab. Often, it will work there.



